frontend/src/tabs/zoom.pug

```diff
@@ -62,8 +62,28 @@
         option(v-bind:value='true') Descending
         option(v-bind:value='false') Ascending
       label order
+    label
+      input.mui-checkbox.medium-font(type="checkbox", v-model="shouldShowFileTypes")
+      span.medium-font show file type in commits
 
-  .zoom__day(v-for="day in filteredUser.commits", v-bind:key="day.date")
+  .fileTypes(v-if="shouldShowFileTypes")
+    .checkboxes.mui-form--inline(v-if="fileTypes.length > 0")
+      label(style='background-color: #000000; color: #ffffff')
+        input.mui-checkbox--fileType(type="checkbox", v-model="isSelectAllChecked")
+        span All&nbsp;
+      template(v-for="fileType in fileTypes")
+        label(
+          v-bind:key="fileType",
+          v-bind:style="{\
+                  'background-color': info.zFileTypeColors[fileType],\
+                  'color': this.getFontColor(info.zFileTypeColors[fileType])\
+                  }"
+        )
+          input.mui-checkbox--fileType(type="checkbox", v-bind:id="fileType", v-bind:value="fileType",
```

Comment:
one question: do you need to v-bind `id` and `value`? would `id` and `value` suffice?

Reply:
v-bind:value should be enough. Thanks for notifying